swarm/network/stream/snapshot_sync_test.go

```diff
@@ -147,20 +147,16 @@ func testSyncingViaGlobalSync(t *testing.T, chunkCount int, nodeCount int) {
 	//array where the generated chunk hashes will be stored
 	conf.hashes = make([]storage.Address, 0)
 
-	err := sim.UploadSnapshot(fmt.Sprintf("testing/snapshot_%d.json", nodeCount))
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	ctx, cancelSimRun := context.WithTimeout(context.Background(), 2*time.Minute)
+	ctx, cancelSimRun := context.WithTimeout(context.Background(), 3*time.Minute)
 	defer cancelSimRun()
 
-	if _, err := sim.WaitTillHealthy(ctx); err != nil {
+	filename := fmt.Sprintf("testing/snapshot_%d.json", nodeCount)
```

Comment:
I'd inline this in the line below

Reply:
what is the purpose? to save a line?
now it's much more clear, IMHO.

Reply:
yes you are correct, its more clear, but not uniform since in the delta below this one it is inlined. choose either one for consistency, i don't mind either one